homeassistant/components/swiss_public_transport/__init__.py

```diff
@@ -11,18 +11,32 @@
 from homeassistant import config_entries, core
 from homeassistant.const import Platform
 from homeassistant.exceptions import ConfigEntryError, ConfigEntryNotReady
-from homeassistant.helpers import device_registry as dr, entity_registry as er
+from homeassistant.helpers import (
+    config_validation as cv,
+    device_registry as dr,
+    entity_registry as er,
+)
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
 
 from .const import CONF_DESTINATION, CONF_START, CONF_VIA, DOMAIN, PLACEHOLDERS
 from .coordinator import SwissPublicTransportDataUpdateCoordinator
 from .helper import unique_id_from_config
+from .services import setup_services
 
 _LOGGER = logging.getLogger(__name__)
 
 
 PLATFORMS: list[Platform] = [Platform.SENSOR]
 
+CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
```

Comment:
```suggestion
CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
```

Reply:
What is the difference @gjohansson-ST ?

Reply:
The one you set allows YAML setup (while it wouldn't do anything). The one I adjust to is going to tell the user to remove their yaml and use UI to setup the integration so it's really only down to informing the user that they can't configure the integration with yaml.

Reply:
oh cool, thanks